The-SMIRNOFF-force-field-format.md

```diff
@@ -450,6 +529,7 @@ So use generics sparingly unless it is your intention to provide generics that s
 ## Version history
 
 ### 1.0
+**QUESTION:** Should this spec be 1.0 or something earlier (e.g. 0.2)?
```

Comment:
Ambivalent. I guess I'd go for 1.0 since it's going to be a core/key initial release that should be stable.

Reply:
no strong opinion.